src/services/alert/MessageProvider/status/index.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable-next-line */
+const dotenv = require('dotenv').config({ path: `${__dirname}/.env` });
+const request = require('request-promise-native');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'request-promise-native'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4499680)